wrudf/wrudf-cmnd.c

```diff
@@ -1037,7 +1038,7 @@ int lscCommand(void) {
 	    fe->permissions & FE_PERM_O_EXEC ? 'x' : '-',
 	    fe->permissions & FE_PERM_O_CHATTR ? 'a' : '-',
 	    fe->permissions & FE_PERM_O_DELETE ? 'd' : '-',
-	    fe->fileLinkCount, (uint32_t)fe->informationLength, filename);
+	    fe->fileLinkCount, (uint64_t)fe->informationLength, filename);
```

Comment:
Cast is not needed. Member `informationLength` is already of `uint64_t` type.

Also no need to create two pull requests for one change. Just update current pull request (use commands `git commit --amend -a` and `git push --force`)

Reply:
got it, thanks!